src/lib/key_gen/sample_keypairs.ml

```diff
@@ -0,0 +1,43 @@
+open Core_kernel
+open Signature_lib
+
+let kps =
+  lazy
+    (let n = 1200 in
+     let generated_keypairs =
+       let sks =
+         Quickcheck.(
+           random_value ~seed:(`Deterministic "Coda_sample_keypairs")
+             (Generator.list_with_length n Private_key.gen))
+       in
+       List.map sks ~f:Keypair.of_private_key_exn
+     in
+     List.cons
+       (* FIXME #2936: remove this "precomputed VRF keypair" *)
+       (* This key is also at the start of all the release ledgers. It's needed to generate a valid genesis transition *)
+       (Keypair.of_private_key_exn
+          (Private_key.of_base58_check_exn
+             "EKFKgDtU3rcuFTVSEpmpXSkukjmX4cKefYREi6Sdsk7E7wsT7KRw" ) )
+       generated_keypairs )
```

Comment:
This isn't used in many places, and is mostly used to grab the added private key. It would be better to separate it like
```ocaml
let genesis_winner =
  Keypair.of_private_key_exn
    (Private_key.of_base58_check_exn
       "EKFKgDtU3rcuFTVSEpmpXSkukjmX4cKefYREi6Sdsk7E7wsT7KRw" )
```
and then use that in the place of this, so that we don't regularly pay the cost of generating 1200 fresh keypairs.